lab-anthony/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "lab-anthony-09-11",
+  "version": "0.1.0",
+  "description": "labs 09-10-11",
+  "main": "server.js",
+  "directories": {
+    "test": "test"
+  },
+  "dependencies": {
+    "body-parser": "^1.15.1",
+    "express": "^4.13.4",
+    "gulp": "^3.9.1",
```

Comment:
You've got this as a dependency and a dev dependency. Which do you think is correct?